packages/styles/dark-theme.css

```diff
@@ -0,0 +1,77 @@
+.cauldron--theme-dark,
+.theme-dark {
```

Comment:
Sorry, I may not have made myself clear when we spoke in slack.

Components in cauldron should only support the `.cauldron--theme-dark` scoped class. `.theme-dark` is something local to the extension, so in the extension the theme context will need to set both classes for backwards compatibility support, but cauldron itself should be agnostic to that.